modules/smart-agent_health-checker/conf/00-health-checker-value.yaml

```diff
@@ -0,0 +1,14 @@
+module: health-checker
+name: value
+id: health_checker_value
```

Comment:
I would remove this to keep the variables names shorter with `value` only.

Reply:
done